pkgs/by-name/we/weevely/package.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, fetchFromGitHub
+, python3
+, python3Packages
+, makeWrapper
+, installShellFiles
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "weevely";
+  version = "4.0.1-unstable-2023-06-18";
```

Comment:
Can't see at all where this `unstable` and date is coming from. I think it should be just `4.0.1`

Reply:
I will use 4.0.2

Reply:
But why should you package an unstable version? Wouldn't it make more sense to use the latest version?

Reply:
> But why should you package an unstable version? Wouldn't it make more sense to use the latest version?

Read the latest commit. It is a good fiix to have.

Reply:
In my opinion you should ask epinna to create a new release then. Where should this end? Creating a new package update pull request every single time he commits anything to this project?

Reply:
Imho there is no need. But if you would like to ask you to the dev, and it accepts, I will update it to 4.0.3.

Reply:
To then use the latest commit again, when he pushes some new commits? Don't see the point